src/watchman/models.clj

```diff
@@ -206,12 +218,38 @@
     (k/set-fields fields)
     (k/where {:id id})))
 
+(defn snooze-msg [snooze-until]
+  "Given a Joda DateTime, return a formatted message for the UI indicating until when this role is snoozed.
+  Formatted string will look like: 'Snoozed until 04-21-15 23:38 PST8PDT'.  Currently PST8PDT is hardcoded in
+  for the timezone."
+  (let [tz (time-core/time-zone-for-id "PST8PDT")
+        formatter (time-format/with-zone (time-format/formatter "MM-dd-yy HH:mm") tz)
+        formatted-time (->> snooze-until
+                            time-coerce/to-date-time
+                            (time-format/unparse formatter))]
+      (str "Snoozed until " formatted-time " " (.getID tz))))
+
+(defn role-snoozed?
+  "Determine if the specified role is still asleep as specified by a snooze. cur-time is provided as an
+  argument so that this result can be made consistent with other timing related queries"
+  ([role] (role-snoozed? role (time-core/now)))
+  ([role cur-time] (let [snooze-until (-?> (:snooze_until role)
```

Comment:
For multi-arity functions like this, have the arg vector on its own line just like for single arity functions.